pom.xml

```diff
@@ -40,6 +40,8 @@
     <spotless.check.skip>false</spotless.check.skip>
     <ban-junit4-imports.skip>false</ban-junit4-imports.skip>
     <hpi.strictBundledArtifacts>true</hpi.strictBundledArtifacts>
+    <hpi.bundledArtifacts>jackson-annotations,jackson-core,jackson-databind</hpi.bundledArtifacts>
```

Comment:
This is unnecessary. The Jenkins ecosystem prefers to depend on "api plugins" that provide libraries rather than each plugin bundling its own libraries. Remove this in favor of org.jenkins-ci.plugins:jackson2-api.